<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Сообщение:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="MessageEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>21</width>
              <height>23</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="CloseClient">
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Отключить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BanClient">
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Бан</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SendMessage">
            <property name="text">
             <string>Отправить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PrivateMessage">
            <property name="text">
             <string>Приват</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="TextViwe">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTextBrowser" name="AllMessages"/>
        <widget class="QTextBrowser" name="PrivateMessages"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="CreatNewWindow"/>
    <addaction name="LoginAdministration"/>
    <addaction name="separator"/>
    <addaction name="Quit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="CreatNewWindow">
   <property name="text">
    <string>Создание нового клиента</string>
   </property>
  </action>
  <action name="LoginAdministration">
   <property name="text">
    <string>Войти администратором</string>
   </property>
  </action>
  <action name="Quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
